scripts/scil_snr_in_roi.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Script to compute signal to noise ratio (SNR) in a region of interest (ROI)
+of a DWI volume. 
+
+It will compute the SNR for all DWI volumes of the input image seperately.
+The output will contain the SNR 
+The mean of the signal is computed inside the mask.
+The standard deviation of the noise is estimated inside noise_mask.
+If it's not supplied, it will be estimated using the data outside medotsu.
+
+If verbose is True, the SNR for every DWI volume will be outputed.
+"""
+
+import numpy as np
+import nibabel as nib
+import matplotlib.pyplot as plt
+
+import os
+import argparse
```

Comment:
put os and argparse before numpy and co 
datetime with os and argparse

dipy with numpy and nibabel

scilpy alone

alphabetical order